src/rpc/RPCHelpers.cpp

```diff
@@ -1703,4 +1711,4 @@ traverseTransactions(
     return response;
 }
 
-}  // namespace RPC
+}  // namespace RPC
```

Comment:
You may need to setup VSCode (or the IDE you are using) to automatically add a newline to the end of any files you change.